src/main.js

```diff
@@ -61,57 +64,63 @@ function makeIdeaCard(ideaCards){
 		`
 	}
 }
-function deleteOrFavorite(){
-	if(event.target.classList.contains('star')){
+
+function deleteOrFavorite(event) {
+	if (event.target.classList.contains('star')) {
 		changeStar(event);
-	}else{
+	} else {
 		deleteIdeaCard(event);
 	}
 }
-function deleteIdeaCard(event){
+
+function deleteIdeaCard(event) {
 	event.preventDefault();
 	var getId = event.target.closest('.delete').id;
 	var newGetId = parseInt(getId);
 	for (var i = 0; i < saveIdeaCards.length; i++) {
-		if(newGetId === saveIdeaCards[i].id){
-			saveIdeaCards.splice(i,1);
+		if (newGetId === saveIdeaCards[i].id) {
+			saveIdeaCards.splice(i, 1);
 			idea.deleteFromStorage();
 		}
 	}
 	makeIdeaCard(saveIdeaCards);
 }
-function changeStar(event){
+
+function changeStar(event) {
 	var getId = event.target.closest('.star');
 	var newGetId = parseInt(getId.id);
 	for (var i = 0; i < saveIdeaCards.length; i++) {
-		if(newGetId === saveIdeaCards[i].id){
+		if (newGetId === saveIdeaCards[i].id) {
 			idea.updateIdea();
 			idea.saveToStorage();
 		}
 	}
 }
-function showFavIdeaCards(){
-	btmSection.innerHTML ="";
+
+function showFavIdeaCards() {
+	btmSection.innerHTML = "";
 	var favIdeaCards = [];
-	if(showStarredBtn.innerText === "Show All Ideas"){
+	if (showStarredBtn.innerText === "Show All Ideas") {
 		showStarredBtn.innerText = "Show Starred Ideas";
 		makeIdeaCard(saveIdeaCards);
 		return;
 	}
 	for (var i = 0; i < saveIdeaCards.length; i++) {
-		console.log(saveIdeaCards[i].star);
-			if(saveIdeaCards[i].star){
-				showStarredBtn.innerText = "Show All Ideas";
-				favIdeaCards.push(saveIdeaCards[i]);
-			}
+		if(saveIdeaCards[i].star){
+		  showStarredBtn.innerText = "Show All Ideas";
+		  favIdeaCards.push(saveIdeaCards[i]);
 		}
-		makeIdeaCard(favIdeaCards);
+	}
+
+	makeIdeaCard(favIdeaCards);
 }
-function retreiveFromLocalStorage(){
-	btmSection.innerHTML ="";
+
+function retreiveFromLocalStorage() {
+	btmSection.innerHTML = "";
 	var items = JSON.parse(localStorage.getItem("savedIdeas"));
 	for (var i = 0; i < items.length; i++) {
```

Comment:
This is where I was receiving the type error, it's disappeared with your update. Awesome!